src/aks-preview/azext_aks_preview/custom.py

```diff
@@ -2477,3 +2483,128 @@ def _aks_mesh_update(
         return None
 
     return aks_update_decorator.update_mc(mc)
+
+def aks_mesh_get_mesh_config(cmd, client, resource_group_name, name):
+    from azure.cli.core import __version__ as local_version
+    from packaging.version import parse
+    from pprint import pprint
+    
+    print('---')
+    contexts, active_context = kubernetes.config.list_kube_config_contexts()
+    if not contexts:
+        print("No context in the kube config file.")
+        return  
+    for item in contexts:
+        if(item['context']['cluster'] == name):
+            cur_context = item['name']
+        
```

Comment:
```suggestion
        if(item['context']['cluster'] == name):
            cur_context = item['name']
            break
```
it's best to break out of the for loop once you've found what you're looking for to save the extra work